version: '3.7'
services:
  standalone_tts_core:
    image: standalone_tts_core
    ports:
      - 42064:42064
    environment:
      - ISDOCKER=cat
    networks:
      - praxisbot
    volumes:
      - "../logs:/Praxis/logs"
      - "../credentials:/Praxis/standalone_tts_core/credentials"
      - "../config.py:/Praxis/standalone_tts_core/config.py"
networks:
  praxisbot:
    name: praxisbot